cpp/src/Slice/Parser.cpp

```diff
@@ -3053,7 +3053,6 @@ Slice::InterfaceDef::allBases() const
 {
     InterfaceList result = _bases;
     result.sort(containedCompare);
-    result.unique(containedEqual);
```

Comment:
It is illegal to list the same interface multiple times in an inheritance list, so there's no point to calling `unique` on `_bases` here.